ctapipe/tools/dl1_merge.py

```diff
@@ -0,0 +1,359 @@
+    output_path = traits.Path(help="merged-DL1 output filename").tag(config=True)
+    skip_images = traits.Bool(
+        help="skip /dl1/event/telescope/images and "
+        "/simulation/event/telescope/images data in output",
+        default_value=False,
+    ).tag(config=True)
+    skip_simu_images = traits.Bool(
+        help="skip /simulation/event/telescope/images " "data in output",
+        default_value=False,
+    ).tag(config=True)
+    skip_parameters = traits.Bool(
+        help="skip /dl1/event/telescope/parameters and "
+        "/simulation/event/telescope/parameters "
+        "data in output",
+        default_value=False,
+    ).tag(config=True)
+    overwrite = traits.Bool(help="overwrite output file if it exists").tag(config=True)
+    progress_bar = traits.Bool(help="show progress bar during " "processing").tag(
+        config=True
+    )
+    file_pattern = traits.Unicode(
+        default_value="*.h5", help="Give a specific file pattern for the" "input files"
+    ).tag(config=True)
+
+    parser = ArgumentParser()
+    parser.add_argument("input_files", nargs="*", type=Path)
+
+    aliases = {
+        "input-dir": "MergeTool.input_dir",
+        "output": "MergeTool.output_path",
+        "pattern": "MergeTool.file_pattern",
+    }
+
+    flags = {
+        "skip-images": (
+            {"MergeTool": {"skip_images": True}},
+            "Skip DL1/Event/Telescope and Simulation/Event/Telescope images in output",
+        ),
+        "skip-simu-images": (
+            {"MergeTool": {"skip_simu_images": True}},
+            "Skip Simulation/Event/Telescope images in output",
+        ),
+        "skip-parameters": (
+            {"MergeTool": {"skip_parameters": True}},
+            "Skip DL1/Event/Telescope parameters in output",
+        ),
+        "overwrite": (
+            {"MergeTool": {"overwrite": True}},
+            "Overwrite output file if it exists",
+        ),
+        "progress": (
+            {"MergeTool": {"progress_bar": True}},
+            "show a progress bar during event processing",
+        ),
+    }
+
+    def setup(self):
+        # prepare output path:
+        self.output_path = self.output_path.expanduser()
+        if self.output_path.exists():
+            if self.overwrite:
+                self.log.warning(f"Overwriting {self.output_path}")
+                self.output_path.unlink()
+            else:
+                self.log.critical(
+                    f"Output file {self.output_path} exists, "
+                    "use `--overwrite` to overwrite"
+                )
+                sys.exit(1)
+
+        PROV.add_output_file(str(self.output_path))
+
+        if self.skip_parameters is True and self.skip_images is True:
+            self.log.warning("Skip-parameters and skip-images are both" "set to True")
+
+        # Get input Files
+        args = self.parser.parse_args(self.extra_args)
+        self.input_files = args.input_files
```

Comment:
I think to make it work with paths given in the config file, this also needs to be `extend(args.input_files)`